packages/vite/src/node/plugins/asset.ts

```diff
@@ -23,6 +23,7 @@ import { cleanUrl, getHash, normalizePath } from '../utils'
 import { FS_PREFIX } from '../constants'
 
 export const assetUrlRE = /__VITE_ASSET__([a-z\d]{8})__(?:\$_(.*?)__)?/g
+export const publicAssetUrlRE = /__VITE_PUBLIC_ASSET__([a-z\d]{8})__/g
```

Comment:
same here, is this being done by the format tool?

Reply:
> same here, is this being done by the format tool?

no, i did this, i found that there are some places that the variables are declared after used, I known that these variables are used in function , but it's a little weird to read.

Reply:
Let's revert these ones, we shouldn't merge unrelated changes to this PR. It is harder to read the history later. Independent of that, I don't think we should move (same history reasons).
Could you merge main @PengBoUESTC, so we can run ecosystem-ci again and check Marko/VitePress?

Reply:
OK